packages/playwright-core/src/utils/hostPlatform.ts

```diff
@@ -86,6 +86,13 @@ function calculatePlatform(): { hostPlatform: HostPlatform, isOfficiallySupporte
         return { hostPlatform: ('ubuntu22.04' + archSuffix) as HostPlatform, isOfficiallySupportedPlatform: false };
       return { hostPlatform: ('ubuntu24.04' + archSuffix) as HostPlatform, isOfficiallySupportedPlatform: false };
     }
+    // Zorin is ubuntu-based but does not have the same versions
+    if (distroInfo?.id === 'zorin') {
+      const zorinMajor = parseInt(distroInfo.version, 10);
+      if (zorinMajor < 17)
```

Comment:
Looking at [this](https://en.wikipedia.org/wiki/Zorin_OS) it seems it should be Ubuntu 22.04 for 17.1 and Ubuntu 24.04 for 17.2.